plugins/textpattern.md

```diff
@@ -31,8 +31,9 @@ This option allows configuring the text patterns that get matched by the `textpa
 
 There are three types of patterns: `inline`, `block`, and `replacement` patterns. Inline patterns have a `start` and an `end` text pattern whereas the block and replacement patterns only have a `start`. A user can specify the formats to be applied to the selection, commands to be executed, or text to be replaced.
 
-When using list commands make sure that the [lists plugin]({{ site.baseurl }}/plugins/lists) is registered for normalized behavior across browsers.
+> **Note**: When using list commands make sure that the [lists plugin]({{ site.baseurl }}/plugins/lists) is registered for normalized behavior across browsers.
 
+&nbsp;
```

Comment:
```suggestion
```
Not needed. I wish it was this easy to concat notes

Reply:
When I actually remove this, the two notes will look like this: 

![Screen Shot 2020-10-22 at 1 29 30 pm](https://user-images.githubusercontent.com/6000860/96821119-a5761400-146a-11eb-9497-bb55e3286e8b.png)

Reply:
as one comes after the other:

![Screen Shot 2020-10-22 at 1 30 52 pm](https://user-images.githubusercontent.com/6000860/96821216-d22a2b80-146a-11eb-94ed-5711c03f3d7e.png)

Reply:
Wait... got it working: 

![Screen Shot 2020-10-22 at 1 32 44 pm](https://user-images.githubusercontent.com/6000860/96821392-1d443e80-146b-11eb-8457-f4c7e4071df6.png)

![Screen Shot 2020-10-22 at 1 33 19 pm](https://user-images.githubusercontent.com/6000860/96821420-29300080-146b-11eb-813c-b85ede63e7a0.png)